Makefile

```diff
@@ -18,6 +18,7 @@ MIGRATION_PLANNER_AUTH ?= local
 PERSISTENT_DISK_DEVICE ?= /dev/sda
 INSECURE_REGISTRY ?= "true"
 DOWNLOAD_RHCOS ?= true
+RHCOS_PASSWORD ?= '$$y$$j9T$$hUUbW8zoB.Qcmpwm4/RuK1$$FMtuDAxNLp3sEa2PnGiJdXr8uYbvUNPlVDXpcJim529'
```

Comment:
if the user doesn't want to set up a password, it will default to this. Is this the right behavior?